### DISCUSSION TITLE
ProxBox Plugin - class 'django.template.exceptions.TemplateDoesNotExist'

### DISCUSSION THREAD

**Discussion by RashitVoroshilov** [netbox-community/netbox#10794] [Q&A] [answered]
Hi, everyone!
Unexpected trouble with ProxBox plugin.

<class 'django.template.exceptions.TemplateDoesNotExist'>

footer.html

Python version: 3.9.7
NetBox version: 3.3.6

Can you help, me where I must dig to resolve this problem?
I understand, that there's a similar discussion, but in that case problem was resolved by updating netbox. My netbox has latest version.

**Comment by candlerb** [comment, marked as answer]
I've never heard of ProxBox before, but I found its project page via a google search, and I found this on the front page:
https://github.com/netdevopsbr/netbox-proxbox#example-of-error-if-not-configuring-the-code-above-on-settingspy

**Reply by RashitVoroshilov** [reply]
Yep, I've find it out tomorrow, I'v just forget about adding two strings to the settings.py:

PROXBOX_TEMPLATE_DIR = BASE_DIR + '/netbox-proxbox/netbox_proxbox/templates/netbox_proxbox'
 'DIRS': [TEMPLATES_DIR, PROXBOX_TEMPLATE_DIR],
 
 Now it works.
